### DISCUSSION TITLE
When inputting backend options to rioxarray, how is it entered?

### DISCUSSION THREAD

**Discussion by andhuang-CLGX** [corteva/rioxarray#409] [General]
E.g.
```
OVERVIEW_RESAMPLING=[NEAREST/AVERAGE/BILINEAR/CUBIC/CUBICSPLINE/LANCZOS]: (since GDAL 3.2) Resampling method used for overview generation. For paletted images, NEAREST is used by default, otherwise it is CUBIC. This overrides, for overview generation, the value of RESAMPLING if it specified.
```
from https://gdal.org/drivers/raster/cog.html#raster-cog

Wondering about the capitalization; is it:
`ds.rio.to_raster("test.tif", driver="COG", tiled=True, lock=SerializableLock(), num_threads=8, overview_resampling="NEAREST")`

or
`ds.rio.to_raster("test.tif", driver="COG", tiled=True, lock=SerializableLock(), num_threads=8, overview_resampling="nearest")`

Thanks

**Comment by snowman2** [comment]
I think that either one should work as it is case insensitive. But, you can try it out and see what happens.

**Reply by andhuang-CLGX** [reply]
Yep you are correct

**Comment by RichardScottOZ** [comment]
Capitalised I think is good for occasions when you are using at least for me, more likely to think - right, GDAL option HERE!
